app/lib/finder_api.rb

```diff
@@ -72,15 +71,24 @@ def sort_batch_results(results)
     case @order
     when 'most-viewed'
       results.sort_by { |hash| hash['popularity'] }.reverse
-    when 'updated-oldest'
-      results.sort_by { |hash| hash['public_timestamp'] }
+    when 'updated-newest' 
+      results.sort_by { |hash| hash['public_timestamp'] }.reverse
     when 'a-z'
       results.sort_by { |hash| hash['title'] }
     else
-      results.sort_by { |hash| hash['public_timestamp'] }.reverse
+      sort_by_relevance(results)
     end
   end
 
+  def sort_by_relevance(results)
+    return results unless relevance_scores_exist?(results)
```

Comment:
Can we make it obvious that "results" are the default,raw(?) results returned by rummager?

Reply:
If the `results` being passed between the methods is the same object, perhaps we should make the attribute readable when it is first assigned?